public/js/views/AllRatingsView.js

```diff
@@ -0,0 +1,33 @@
+/* global TFT */
+
+"use strict";
+
+var Backbone = require("backbone");
+var RatingItemView = require("./RatingItemView");
+
+module.exports = Backbone.View.extend({
+	className: "allRatingsList",
+	template: TFT.ratings,
```

Comment:
your template file is allRatings, not ratings, and this is causing an error that breaks your application as soon as you navigate to the ratings in your app